.pipelines/cni/cilium/cilium-scale-test.yaml

```
@@ -66,7 +66,9 @@ stages:
                 export CILIUM_IMAGE_REGISTRY=${CILIUM_IMAGE_REGISTRY}
                 if ${IS_DUALSTACK}; then
                   echo "Use dualstack daemonset for Cilium"
-                  envsubst '${CILIUM_VERSION_TAG},${CILIUM_IMAGE_REGISTRY},${IPV6_HP_BPF_VERSION}' < test/integration/manifests/cilium/v${DIR}/cilium-agent/templates/daemonset-dualstack.yaml | kubectl apply -f -
+                  export IPV6_IMAGE_REGISTRY=acnpublic.azurecr.io
+                  export IPV6_HP_BPF_VERSION=$(make ipv6-hp-bpf-version)
+                  envsubst '${CILIUM_VERSION_TAG},${CILIUM_IMAGE_REGISTRY},${IPV6_IMAGE_REGISTRY},${IPV6_HP_BPF_VERSION}' < test/integration/manifests/cilium/v${DIR}/cilium-agent/templates/daemonset-dualstack.yaml | kubectl apply -f -
```

```suggestion
                  export IPV6_IMAGE_REGISTRY=acnpublic.azurecr.io
                  envsubst '${CILIUM_VERSION_TAG},${CILIUM_IMAGE_REGISTRY},${IPV6_IMAGE_REGISTRY},${IPV6_HP_BPF_VERSION}' < test/integration/manifests/cilium/v${DIR}/cilium-agent/templates/daemonset-dualstack.yaml | kubectl apply -f -
```

scale-test consumes `IPV6_HP_BPF_VERSION` from pipeline vars. We only need to hardcode `IPV6_IMAGE_REGISTRY`